Arm Subsystem Skeleton Creation

TechnototesLaptop opened the pull request:
created the arm subsystem(basic data and commands [i think]) and added it into setup, robot, and hardware

no real values, just a skeleton :>

TechnototesLaptop committed:
10/19/24 - Harshini
created the arm subsystem(basic data and commands [i think]) and added it into setup, robot, and hardware

no real values, just a skeleton :>

kevinfrei commented on Twenty403/src/main/java/org/firstinspires/ftc/twenty403/Hardware.java:
```diff
@@ -28,6 +28,7 @@ public class Hardware implements Loggable {
     public CRServo intake;
     public ColorSensor colorSensor;
     public Rev2MDistanceSensor rev2MDistanceSensor;
+    public EncodedMotor rotate1, rotate2, slides;
```
EncodedMoter<DcMotorEx> here!

kevinfrei commented on Twenty403/src/main/java/org/firstinspires/ftc/twenty403/Hardware.java:
```diff
@@ -55,6 +56,11 @@ public Hardware(HardwareMap hwmap) {
             rev2MDistanceSensor = new Rev2MDistanceSensor(Setup.HardwareNames.DIST1);
         }
+        if (Setup.Connected.ARMSUBSYSTEM) {
+            rotate1 = new EncodedMotor(Setup.HardwareNames.ROTATEMOTOR1);
```
Ditto here: All need <DcMotorEx>

kevinfrei reviewed (changes requested):
Fix up the types

TechnototesLaptop committed:
10/19/24 - Harshini
created the arm subsystem(basic data and commands [i think]) and added it into setup, robot, and hardware

no real values, just a skeleton :>

**for the methods, put both motors together

kevinfrei reviewed (approved):
Booyah!
